dftd3/dftd3.py

```diff
@@ -282,16 +281,16 @@ def d3(
                         d2[0] = r2ab[ij]
                         d2[1] = r2ab[jk]
                         d2[2] = r2ab[ik]
-                        t1 = (d2[0] + d2[1] - d2[2]) / math.sqrt(d2[0] * d2[1])
-                        t2 = (d2[0] + d2[2] - d2[1]) / math.sqrt(d2[0] * d2[2])
-                        t3 = (d2[2] + d2[1] - d2[0]) / math.sqrt(d2[1] * d2[2])
+                        t1 = (d2[0] + d2[1] - d2[2]) / jnp.sqrt(d2[0] * d2[1])
+                        t2 = (d2[0] + d2[2] - d2[1]) / jnp.sqrt(d2[0] * d2[2])
+                        t3 = (d2[2] + d2[1] - d2[0]) / jnp.sqrt(d2[1] * d2[2])
                         ang = 0.375 * t1 * t2 * t3 + 1.0
                         e63 = e63 + tmp * c9 * ang / (d2[0] * d2[1] * d2[2]) ** 1.50
 
     repulsive_abc_term = s6 * e63
     repulsive_abc += repulsive_abc_term
 
-    return attractive_r6_vdw, attractive_r8_vdw, repulsive_abc
+    return attractive_r6_vdw + attractive_r8_vdw #, repulsive_abc
```

Comment:
A-ha! I see now why JAX complains! I think what we want to do is to have 3 functions that compute each of the components of the D3 correction separataly: attractive R6, attractive R8, and 3-body repulsive. Then when asking for derivatives we sum their results.

Reply:
okey, will start working towards that